### PR TITLE
Add new tasks to the repository 

### PR DESCRIPTION
**Pull request by AsfaMumtaz**
Update-cd-repo task added to the repository

### CODE REVIEW

**Comment by stakater-user** [comment]
@AsfaMumtaz Validation Successful

File: clustertask/github-update-cd-repo/templates/clustertask.yaml [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+            TEAM=$(params.team)
+          else
+            TEAM=$(echo $(params.namespace) | cut -d'-' -f 1)
+          fi
+          echo "Team is ${TEAM}"
+          export REPO=$(echo $(params.repoPath) | rev | cut -d'/' -f 1 | rev )
+          echo "Repo is ${REPO}"
+          echo "Adding nexus helm repo $(params.helm_registry)"
+          helm repo add nexus $(params.helm_registry) --username ${HELM_REG_USER} --password ${HELM_REG_PASSWORD}
+
```
Comment (rasheedamir) [inline clustertask/github-update-cd-repo/templates/clustertask.yaml:111]: @AsfaMumtaz too many empty lines here

File: clustertask/github-update-cd-repo/README.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# gitlab update cd repo
```
Comment (rasheedamir) [inline clustertask/github-update-cd-repo/README.md:1]: @AsfaMumtaz is this for gitlab or github?

File: clustertask/github-update-cd-repo/templates/clustertask.yaml [hunk 3]
```diff
@@ -0,0 +1,154 @@
+apiVersion: tekton.dev/v1beta1
+kind: ClusterTask
+metadata:
+  name: {{ include "stakater-github-update-cd-repo.name" . }}
+spec:
+  workspaces:
+    - name: source
+  params:
```
Comment (rasheedamir) [inline clustertask/github-update-cd-repo/templates/clustertask.yaml:8]: we have 3 different naming conventions being followed here for the param names; can we fix it

**Comment by stakater-user** [comment]
@rasheedamir Validation Successful

**Review by rasheedamir — APPROVED** [review]

**Pull request closed without merging**